.github/workflows/ci.yml

```diff
@@ -159,14 +167,15 @@ jobs:
           # Check with all features.
           # (But watch out for backend-selection features in wgpu-core; some of
           # those only build on the right platforms.)
+          cargo clippy --target ${{ matrix.target }} -p wgpu-hal --tests --all-features
+          cargo clippy --target ${{ matrix.target }} -p wgpu-core --tests --all-features
           cargo clippy --target ${{ matrix.target }} -p wgpu -p wgpu-info -p player --tests --all-features
-          cargo clippy --target ${{ matrix.target }} -p wgpu-core --tests --features="portable_features"
 
           # build docs
           # (Watch out for backend-selection features in wgpu-core; some of
           # those only build on the right platforms.)
           cargo doc --target ${{ matrix.target }} -p wgpu -p wgpu-info -p player --all-features --no-deps
-          cargo doc --target ${{ matrix.target }} -p wgpu-core --no-deps --features="portable_features"
+          cargo doc --target ${{ matrix.target }} -p wgpu-core --no-deps --all-features
```

Comment:
Same here

Reply:
Done.

Reply:
Now that i think about it, I could also just use `--workspace`. Wdyt?

Reply:
Now that it should work, let's do it

Reply:
`--workspace` fails to compile something with the `v8` dependency, which is something in Deno I assume, for Android and iOS.
https://github.com/gfx-rs/wgpu/actions/runs/4218266653/jobs/7322645792
https://github.com/gfx-rs/wgpu/actions/runs/4218266653/jobs/7322645995

Reverting again.

Reply:
Done.